### PR TITLE
Admission trial project

### PR DESCRIPTION
**Pull request by momotsow**
For this admission project for microverse, I created a YouTube screen

### CODE REVIEW

**Comment by momotsow** [comment]
STATUS: APPROVED ✔️ 🟢
Hi @momotsow 😄 ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Highlights ✨
✔️ Good job working on the project.
✔️ Linters are passing.

Optional suggestions ☑️
[OPTIONAL]- Change the title in your <head></head> tag

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

**Review by sajid-munawar — COMMENTED** [review]
Hi there, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ⚠️

You have submitted a project, without reviewing it as expected.

- [ ]  [REQUIRED] Based on the requirement for this project, you are expected to use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to leave review comments. Please visit the link above to learn how to do that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

>Invalid Code Review Request **does not count** into the code reviews limit.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @momotsow , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

You have submitted a project, that has not used `GitHub's` Review changes feature, please see this [>> link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) for more information on reviewing your project.
## Tips :bulb: 
- [ ] Assume you are reviewing another student's project
- [ ] Check the student's code base for any HTML and CSS mistakes and make inline comments pointing out these mistakes and how they can be corrected.
- [ ] Remember to use inline comments when making the review, the comments should not be less than 2.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
```
Comment (momotsow) [inline index.html:8]: Change the title in your tag to suit the content of the project

File: index.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        <div class="main-video">
+            <iframe width="660" height="415" src="https://www.youtube.com/embed/B-ytMSuwbf8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="side-videos">
+            <div class="side-video">
+                
+                <iframe width="150" height="100" src="https://www.youtube.com/embed/B-ytMSuwbf8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                <div class="video-details">
+                    <a href="#" class="video-title">Video Title</a>
+                    <a href="#" class="video-channel-name">Channel Name</a>
```
Comment (momotsow) [inline index.html:48]: This is a list of video details, and an <ul> tag with a <li> tag for each video detail would be better appropriate in this situation for semantic reasons. Kindly change the HTML tags.

**Review by momotsow — COMMENTED** [review]
Hi @momotsow 😊,

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
Good overall code structure ✔️
Good use of semantic tags✔️

Required Changes ♻️
- Check the comments under the review.
- Try to use more descriptive commit messages. 😀

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

You're almost there, @momotsow  Just a little bit extra and you'll be done. Goodluck✨
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @momotsow, 

# Required Changes ♻️

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Comments are clear and easy to understand ✔️
- Explain the reason why a change is requested ✔️
- Good using of inline comments ✔️ 
- Correct use of summary ✔️  


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

For this comment: 👉🏽 "Change the title in your tag to suit the content of the project"

- [ ] Please 🙏🏽 instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
- [ ] Please 🙏🏽 remember to be polite and clear in your comments - your job here is to help another person improve in a friendly way.
- [ ] Kindly you can include emojis and gifs to make your code review more human-friendly 👽.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (momotsow) [inline index.html:8]: I think it would be better to give your title in the <title> tag a descriptive name. Kindly change the title tag. 🙏🏽.

**Review by Santiago220991 — APPROVED** [review]
Hi @momotsow , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

:heavy_check_mark: Used GitHub review features.
:heavy_check_mark:  Polite comments.
:heavy_check_mark: Inline comments.

## Optional suggestions
[OPTIONAL] I think is a good idea to add a GIF image in your summary as I did here to make your summary more human friendly. :space_invader: :space_invader:  You can use an extension for chrome called "GIFS for GitHub" to do this.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**